osmoutils/accum/export_test.go

```diff
@@ -1,15 +1,64 @@
 package accum
 
 import (
+	"errors"
+
+	"github.com/gogo/protobuf/proto"
+
 	"github.com/cosmos/cosmos-sdk/store"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/osmoutils"
 )
 
+// GetPosition returns the position associated with the given address.
+// This function is currently used for testing purposes only.
+// If there is a need to use this function in production, it
+// can be moved to a non-test file.
+func (accum AccumulatorObject) GetPosition(addr sdk.AccAddress) Record {
+	position := Record{}
+	osmoutils.MustGet(accum.store, formatPositionPrefixKey(accum.name, addr.String()), &position)
+	return position
+}
+
+// GetAllPositions returns all positions associated with the receiver accumulator.
+// Returns error if any database errors occur.
+// This function is currently used for testing purposes only.
+// If there is a need to use this function in production, it
+// can be moved to a non-test file.
+func (accum AccumulatorObject) GetAllPositions() ([]Record, error) {
+	return osmoutils.GatherValuesFromStorePrefix(accum.store, formatPositionPrefixKey(accum.name, ""), parseRecordFromBz)
+}
```

Comment:
Note: this is a test helper for getting all of the positions of an accumulator

Reply:
I ended up not using it but leaving it in case you need @AlpinYukseloglu . If not, we can remove in a subsequent PR

Reply:
This is super useful for testing edge cases on `AddToPosition` and `RemoveFromPosition` – thanks for adding it!